zuul.d/cisco-nxos-jobs.yaml

```diff
@@ -1,40 +1,32 @@
 ---
-# ansible-collections/cisco.nxos jobs
-
+# ansible-ee
 - job:
-    name: ansible-network-nxos-appliance
-    parent: ansible-network-appliance-base
-    pre-run: playbooks/ansible-network-nxos-appliance/pre.yaml
-    run: playbooks/ansible-network-nxos-appliance/run.yaml
+    name: ansible-ee-integration-cisco-nxos
+    parent: ansible-ee-tests
+    pre-run:
+      - playbooks/ansible-ee/integration/pre.yaml
+      # Ensure we install the bindeps
+      - playbooks/ansible-network-appliance-base/pre.yaml
+      - playbooks/ansible-network-nxos-appliance/pre.yaml
+    run: playbooks/ansible-ee/integration/run.yaml
+    vars:
+      ansible_runner_container_version: latest
+      container_image_tests:
+        - integration
+      test_ansible_ssh_private_key_file: ~/.ssh/id_ecdsa
+      test_ansible_python_interpreter: /usr/bin/python3.8
+      test_ansible_network_cli_ssh_type: paramiko
+      test_fips_mode: false
     host-vars:
       nxos:
         ansible_network_os: nxos
     required-projects:
       - name: github.com/ansible/ansible-zuul-jobs
     nodeset: nxos-9
-
-- job:
-    name: ansible-test-network-integration-nxos
-    abstract: true
-    dependencies:
-      - name: build-ansible-collection
-        soft: true
-    parent: ansible-network-nxos-appliance
-    pre-run:
-      - playbooks/ansible-test-base/pre.yaml
-      - playbooks/ansible-test-network-integration-base/pre.yaml
-    run: playbooks/ansible-test-base/run.yaml
-    post-run:
-      - playbooks/ansible-test-network-integration-base/post.yaml
-      - playbooks/ansible-test-base/post.yaml
-    required-projects:
-      - name: github.com/ansible/ansible
-      - name: github.com/ansible-collections/cisco.nxos
     timeout: 10800
-    vars:
-      ansible_collections_repo: github.com/ansible-collections/cisco.nxos
-      ansible_test_command: network-integration
-      ansible_test_integration_targets: "nxos_.*"
+    files:
+      - ^plugins/.*$
+      - ^tests/integration/.*$
```

Comment:
We should also include requirements.txt to this, I believe. This would make sure that PRs like this https://github.com/ansible-collections/cisco.nxos/pull/447 would also be tested.

Reply:
Indeed, nice catch, we can also add test-requirements.txt.